distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/ServiceManagerImpl.java

```diff
@@ -596,35 +592,6 @@ public void waitForSourcesToBeAvailable(String restPath, String... sources)
     }
 
     @Override
-    public void waitForAllConfigurations() throws InterruptedException {
```

Comment:
❗️ We need to replace this code and wait until all files have their file path attribute set to know that they all have been processed.